frontend/src/components/Explanation.js

```diff
@@ -1 +1,31 @@
-// This is where the explanation component will be created
\ No newline at end of file
+import React from 'react';
+import { Jumbotron} from 'reactstrap';
+const ExplanationImage = '/images/explanation_background.PNG'
+
+
+//Jumbotron component
+const Explanation = (props) => {
+  return (
+    <div>
+      {/* In the following line the jumbotrons style was inlined. In the style, The background image was set as the image that was previously imported, and its size
+             had been set to cover the entire component, which caused the image to be stretched to fit the needed size. */}
+      <Jumbotron style={{ backgroundImage: `url(${ExplanationImage})`, backgroundSize: 'cover'}}>
```

Comment:
Please move this to a stylesheet named Explanation.css and import it into the file.